openhands/resolver/prompts/guess_success/issue-success-check.jinja

```diff
@@ -1,15 +1,15 @@
-Given the following issue description and the last message from an AI agent attempting to fix it, determine if the issue has been successfully resolved.
+Given the following issue description and the last message from an AI agent attempting to fix it, determine if the issue has been successfully resolved based on the changes made and their expected impact. Make your own judgment based on the evidence provided - do NOT defer to or wait for human review.
 
 Issue description:
 {{ issue_context }}
 
 Last message from AI agent:
 {{ last_message }}
 
-(1) has the issue been successfully resolved?
-(2) If the issue has been resolved, please provide an explanation of what was done in the PR that can be sent to a human reviewer on github. If the issue has not been resolved, please provide an explanation of why.
+(1) Based on the changes made and their expected impact, has the issue been successfully resolved?
+(2) Provide a clear explanation of what was done in the PR and how it addresses the issue. Focus on the concrete changes and their expected effects, not on the need for external verification.
```

Comment:
```suggestion
(2) Provide a clear explanation of what was done in the PR and whether it addresses the issue. Focus on the concrete changes and their expected effects, not on the need for external verification.
```
This will probably be more effective if the issue is not resolved.